open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/java/org/odpi/openmetadata/userinterface/uichassis/springboot/api/rex/RepositoryExplorerController.java

```diff
@@ -496,7 +496,7 @@ public RexTraversalResponse rexTraversal(@RequestBody RexTraversalRequestBody bo
         try {
             serverName          = body.getServerName();
             serverURLRoot       = body.getServerURLRoot();
-            enterpriseOption    = body.getEnterpriseOption();
+             enterpriseOption    = body.getEnterpriseOption();
```

Comment:
Fix alignment

Reply:
I think you are looking at an old version of the fix this file is not in the current fix